.circleci/config.yml

```diff
@@ -239,6 +241,9 @@ jobs:
       - run:
           name: run the actual tests
           command: ./mvnw -B org.jacoco:jacoco-maven-plugin:report org.jacoco:jacoco-maven-plugin:report-aggregate clean install -Punit-tests,coverage -ntp
+      - run:
```

Comment:
the review request was removed. That said, you may be on the right track here to report coverage for the unit tests separately from the non-confidential integration tests below. You may need to experiment with the `-F` parameter to see if you can get codecov to merge the results or just keep them reported as separate jobs